src/main/scala/nl/knaw/dans/lib/error/package.scala

```diff
@@ -76,4 +76,145 @@ package object error {
         Success(xs.map(_.get).to(canBuildFrom))
     }
   }
+
+  implicit class FailFastStream[T](val stream: Stream[Try[T]]) {
+    /**
+     * Evaluates a `Stream` of `Try`s into either:
+     *  - one `Success` with a stream of `T`s or
+     *  - a `Failure` with the first exception contained in the stream
```

Comment:
not entirely sure what you mean here. You evaluate a stream of `Try`s. If all are successful you return the stream inside a `Success`. If only one element is a `Failure`, the evaluation is terminated and a `Failure` is returned with the exception that was inside that first `Failure`. I don't really see the point of doing anything with the `Successes` that come prior to this first `Failure`. But maybe I'm overlooking something? Let me know!